lib/curation_concerns/configuration.rb

```diff
@@ -127,6 +127,13 @@ def lock_retry_delay
       @lock_retry_delay ||= 200 # milliseconds
     end
 
+    # @!attribute [w] ingest_queue_name
+    #   ActiveJob queue to handle ingest-like jobs.
+    attr_writer :ingest_queue_name
+    def ingest_queue_name
```

Comment:
Should this also be exposed in `lib/generators/curation_concerns/templates/config/curation_concerns.rb` so folks know they can tweak it in their apps?